modules/serverless-creating-service-deployed-by-openshift-pipelines.adoc

```diff
@@ -0,0 +1,40 @@
+:_content-type: PROCEDURE
+[id="serverless-creating-service-deployed-by-openshift-pipelines_{context}"]
+= Creating a service deployed by {pipelines-shortname}
+
+Using the {ocp-product-title} web console, you can create a service that will be deployed by {pipelines-shortname}.
+
+.Procedure
+
+. In the {ocp-product-title} web console *Developer* perspective, navigate to *Add* and select the *Import from Git* option.
++
+image::pipelines-integration-importing-from-git.png[Importing from Git]
+
+. In the *Import from Git* dialog, specify project metadata by doing the following:
++
+* Specify the Git Repository URL.
+* If necessary, specify the context directory. This is the subdirectory inside the repository that contains the root of application source code.
+* Optionally, specify the application name. By default, the repository name is used.
+* Select the *Serverless Deployment* resource type.
+* Fill the *Add pipeline* checkbox. The pipeline is automatically selected based on the source code and its visualization is shown on the scheme.
+* Specify any other relevant settings.
```

Comment:
Should this be an ordered list?

Reply:
@abhatt-rh It's strictly required, since all these steps are done on the same page. I'd keep it unordered, in case the user wants to jump around and do things in a different order.